### PR TITLE
PCM-7317 - Support flagging a case as "Consultant Engagement in Progress"

### PR DESCRIPTION
**Pull request by anujsi**
@vrathee @gandhikeyur Please review.
https://projects.engineering.redhat.com/browse/PCM-7317

### CODE REVIEW

File: app/cases/controllers/cepModal.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                    CaseService.kase.cep = true;
+                    CaseService.submittingCep = false;
+                    AlertService.addSuccessMessage(gettextCatalog.getString('CEP has been updated successfully'));
+                    angular.copy(CaseService.kase, CaseService.prestineKase);
+                },
+                function (error) {
+                    $scope.showErrorMessage(error);
+                });
+                CaseService.populateComments($stateParams.id).then(function (comments) {
+                    $scope.closeModal();
```
Comment (vrathee) [inline app/cases/controllers/cepModal.js]: Use async await here

File: app/cases/controllers/commonConfirmationModal.js [hunk 2]
```diff
@@ -49,5 +53,23 @@ export default class CommonConfirmationModal {
...
+            CaseService.submittingCep = true;
+            var caseJSON = {'cep': false};
+            var updateCase = strataService.cases.put(CaseService.kase.case_number, caseJSON);
+            updateCase.then(function (response) {
+                CaseService.checkForCaseStatusToggleOnAttachOrComment();
+                AlertService.clearAlerts();
+                AlertService.addSuccessMessage(gettextCatalog.getString('CEP has been updated successfully'));
+                CaseService.kase.cep = false;
+                angular.copy(CaseService.kase, CaseService.prestineKase);
+                CaseService.submittingCep = false;
```
Comment (vrathee) [inline app/cases/controllers/commonConfirmationModal.js]: use async await

**Comment by anujsi** [comment]
@vrathee I have made changes as per your suggestion. Please review.

**Comment by vrathee** [comment]
Reviewed!

**Pull request merged**